devel/libtapi/Portfile

```diff
@@ -5,23 +5,23 @@ PortGroup               cmake 1.1
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               github 1.0
 
-github.setup            tpoechtrager apple-libtapi 664b84
+github.setup            tpoechtrager apple-libtapi b7b5bdbfda9e8062d405b48da3b811afad98ae76
```

Comment:
Just a nitpick, I just find the shorter commit easier to read over the longer ones.

Doesn't really make any difference